adventure/api.py

```diff
@@ -64,4 +64,10 @@ def move(request):
 @api_view(["POST"])
 def say(request):
     # IMPLEMENT
-    return JsonResponse({'error':"Not yet implemented"}, safe=True, status=500)
+    player = request.user.player
+    player_id = request.user.player.id
+    room = player.room()
+    currentPlayerUUIDs = room.playerUUIDs(player_id)
+    for p_uuid in currentPlayerUUIDs:
+        pusher.trigger(f'p-channel-{p_uuid}', u'broadcast', {'message': f'<{player.user.username}> says "{message}"'})
```

Comment:
I think this sends a message to the player who performs the say command, yes? You might consider how it could send a message to everyone 'except' the player who performed the command. This is one way in which it might be differentiated from shout. Of course, shout also sends messages to players even if they are not in the room so that is another difference.